src/settingsview.cpp

```diff
@@ -30,7 +30,12 @@ SettingsView::SettingsView(QWidget *parent)
     connect(ui->moveToTrashToggle, &QCheckBox::clicked, this, &SettingsView::setMoveToTrash);
     connect(ui->reopenTabToggle, &QCheckBox::clicked, this, &SettingsView::setReopenTab);
     connect(ui->browseButton, &QPushButton::clicked, this, &SettingsView::browseDownloadDir);
-    connect(ui->downloadDirPathCopy, &QPushButton::clicked, this, &SettingsView::copyDownloadPathToClipboard);
+    connect(ui->downloadDirPathCopy, &QPushButton::clicked, [this]() {
+        copySettingsPathToClipboard(ui->downloadDirPathCopy, KiwixApp::instance()->getSettingsManager()->getDownloadDir());
```

Comment:
Too long line. Introduce a local variable `settingsMgr` and use it whereever possible in this function.

Reply:
I see that the *whereever possible in this function* part had to be emphasized.

Reply:
Yeah sorry, I missed committing this chunk of the constructor changes